webapp/crashstats/crashstats/tests/test_jinja_helpers.py

```diff
@@ -204,66 +180,24 @@ def test_truncate_short_desc(self):
         assert len(qs["short_desc"][0]) == 255
         assert qs["short_desc"][0].endswith("...")
 
-    def test_corrected_os_version_name(self):
-        report = self._create_report(
-            os_name="Windoooosws",
-            os_pretty_version="Windows 10",
-            crashing_thread=self.CRASHING_THREAD,
-        )
-        url = generate_create_bug_url(
-            f"http://localhost:8000/report/index/{self.CRASH_ID}",
-            self.TEMPLATE,
-            report,
-        )
-        qs = self._extract_query_string(url)
-        assert qs["op_sys"] == ["Windows 10"]
-
-        # os_name if the os_pretty_version is there, but empty
-        report = self._create_report(
-            os_name="Windoooosws",
-            os_pretty_version="",
-            crashing_thread=self.CRASHING_THREAD,
-        )
-        url = generate_create_bug_url(
-            f"http://localhost:8000/report/index/{self.CRASH_ID}",
-            self.TEMPLATE,
-            report,
-        )
-        qs = self._extract_query_string(url)
-        assert qs["op_sys"] == ["Windoooosws"]
-
-        # "OS X <Number>" becomes "macOS"
-        report = self._create_report(
-            os_name="OS X",
-            os_pretty_version="OS X 11.1",
-            crashing_thread=self.CRASHING_THREAD,
-        )
-        url = generate_create_bug_url(
-            f"http://localhost:8000/report/index/{self.CRASH_ID}",
-            self.TEMPLATE,
-            report,
-        )
-        qs = self._extract_query_string(url)
-        assert qs["op_sys"] == ["macOS"]
-
-        # "Windows 8.1" becomes "Windows 8"
-        report = self._create_report(
-            os_name="Windows NT",
-            os_pretty_version="Windows 8.1",
-            crashing_thread=self.CRASHING_THREAD,
-        )
-        url = generate_create_bug_url(
-            f"http://localhost:8000/report/index/{self.CRASH_ID}",
-            self.TEMPLATE,
-            report,
-        )
-        qs = self._extract_query_string(url)
-        assert qs["op_sys"] == ["Windows 8"]
-
-        # "Windows Unknown" becomes plain "Windows"
+    @pytest.mark.parametrize(
+        "os_name, os_pretty_version, op_sys",
+        [
+            ("Windoooosws", "Windows 10", "Windows 10"),
+            # os_name if the os_pretty_version is there, but empty
+            ("Windoooosws", "", "Windoooosws"),
+            # "OS X <Number>" becomes "macOS"
+            ("OS X", "OS X 11.1", "macOS"),
+            # "Windows 8.1" becomes "Windows 8"
+            ("Windows NT", "Windows 8.1", "Windows 8"),
+            # "Windows Unknown" becomes plain "Windows"
+            ("Windows NT", "Windows Unknown", "Windows"),
+        ],
+    )
+    def test_corrected_os_version_name(self, os_name, os_pretty_version, op_sys):
```

Comment:
I cleaned up these tests to use `pytest.mark.parametrize` because it's some boilerplate repeated a whole bunch of times.

Reply:
Much cleaner